cmd/eulabia-sensor/main.go

```diff
@@ -1,9 +1,63 @@
 package main
 
 import (
-	"github.com/greenbone/eulabeia"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/google/uuid"
+	"github.com/greenbone/eulabeia/connection"
+	"github.com/greenbone/eulabeia/connection/mqtt"
+	"github.com/greenbone/eulabeia/messages"
+	"github.com/greenbone/eulabeia/messages/handler"
+	"github.com/greenbone/eulabeia/sensor/handler/memory"
 )
 
 func main() {
-	print(eulabeia.Hello())
+	topic := "greenbone.sensor"
+	server := flag.String("server", "localhost:1883", "A clientid for the connection")
```

Comment:
There will most likely be a config files, environment variable mix. But unless we have the need we don't implement it. So Although it will most likely change currently we choose the easiest option.

Reply:
We will most likely endup with a mix of env variables and configuration file; but currently we don't need it because we have little to no configuration. We will do that when we start with openvas integration since then the structure of the configuration file will be more clear.

Reply:
There is an own task for it; hence I close it.